openapi/components/schemas/AML.yaml

```
@@ -2,22 +2,24 @@ type: object
 properties:
   firstName:
     type: string
-    description: First Name.
+    description: First name of the individual, or name of entity.
     readOnly: true
     example: Benjamin
   lastName:
     type: string
-    description: Last Name. `null` if it is a single-name entity.
+    description: >-
```

```suggestion
    description: |-
```

I thought we should use `|-` according to https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md?

---

Excellent point! I updated all references in [f0ae0a1](https://github.com/Rebilly/api-definitions/pull/983/commits/f0ae0a1851739d33b831de00b99996808176cb2d)